spec/rubocop/cop/style/send_spec.rb

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::Send do
+  subject(:cop) { described_class.new }
+
+  context 'with send' do
+    after(:each) { expect(cop.offenses.size).to eq(1) }
+
+    context 'and with a receiver' do
+      it 'registers an offense for an invocation with args' do
+        inspect_source(cop, 'Object.send(:inspect)')
+      end
+
+      it 'registers an offense for an invocation without args' do
```

Comment:
Btw, why are we registering an offence for `send` without arguments? That would clearly be a false positive.  We should change this.